src/main/java/ch/uzh/ifi/hase/soprafs23/rest/dto/SettingsPutDTO.java

```diff
@@ -1,15 +1,30 @@
-/*package ch.uzh.ifi.hase.soprafs23.rest.dto;
+package ch.uzh.ifi.hase.soprafs23.rest.dto;
 
+import ch.uzh.ifi.hase.soprafs23.constant.aTopic;
 import ch.uzh.ifi.hase.soprafs23.custom.Settings;
 
 public class SettingsPutDTO {
-    private Settings settings;
+    private int rounds;
+    private int roundTime;   // in seconds
+    private aTopic aTopic;
 
-    public Settings getSettings() {
-        return settings;
+    public int getRounds() {
+        return rounds;
     }
 
-    public void setSettings(Settings settings) {
-        this.settings = settings;
+    public int getRoundTime() {
+        return roundTime;
     }
-}*/
+
+    public aTopic getaTopic() {
```

Comment:
Here too, the naming is very odd. In case this is because of issues that arise in the DTOMapper, I think I know a proper solution.